elki/src/main/java/ch/ethz/globis/pht/pre/EmptyPP.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2011-2015 ETH Zurich. All Rights Reserved.
+ *
+ * This software is the proprietary information of ETH Zurich.
+ * Use is subject to license terms.
+ */
+package ch.ethz.globis.pht.pre;
+
+import java.util.Arrays;
+
+import ch.ethz.globis.pht.pre.ColumnType.DoubleColumn;
+import ch.ethz.globis.pht.pre.ColumnType.FloatColumn;
+import ch.ethz.globis.pht.pre.ColumnType.IntColumn;
+import ch.ethz.globis.pht.pre.ColumnType.LongColumn;
+import ch.ethz.globis.pht.util.BitTools;
+
+public class EmptyPP implements PreProcessorPoint {
+
+	private ColumnType[] types;
+	
+	public EmptyPP() {
+		types = null;
+	}
+	
+	public EmptyPP(ColumnType[] types) {
+		this.types = Arrays.copyOf(types, types.length);
+	}
+	
+	@Override
+	public void pre(double[] raw, long[] pre) {
+		if (types != null) {
+			for (int d=0; d<raw.length; d++)
```

Comment:
I fully agree with using {} everywhere. This class was implemented by a student who wasn't aware of that. Let me know if you see any other missing {}.